src/client/auth/abstractAuthentication.ts

```diff
@@ -19,6 +19,20 @@ export interface AuthenticationMethod {
 	): Promise<AuthenticationResult>
 }
 
+export interface AuthenticationMethodUN {
+	TYPE: string
+	getTokenProof(request: AuthenticateInterface): Promise<AuthenticationResult>
+}
+
+export interface MultiAuthenticateInterface {
+	options?: any
```

Comment:
@nicktaras are you going to add additional properties to that interface? if no then better avoid "options" and describe types if its possible

Reply:
Yes, the intention is to move global settings for multi-tokens into this param / interface in the next steps of the epic.

Reply:
in case if you are going to add additional data as properties of **options** then this interface is redundant. 
it ok if some inteface have struture like {options, data, config} . is it our case?
and please update type of **options** from any to some specific. much easier to read/review code when exact types used.

Reply:
I removed this interface, we will need to add a type for options but this will be the same for all tokens in most cases.